index.js

```diff
@@ -88,6 +88,14 @@ const register = (server, pluginOptions) => {
         const txt = pages.map(url => `${protocol}://${request.info.host}${url.path}`).join('\n');
         return h.response(txt).type('text/plain');
       }
+      // assume .json:
+      if (request.query.meta) {
+        return pages.map(page =>
```

Comment:
why are we doing this?

Reply:
Okay that was dumb, I started out selectively adding the fields and then after realizing we didn't need to do that, didn't fix it back.